### PR TITLE
Api add locale product projection

### PR DESCRIPTION
**Pull request by barbara79**

### CODE REVIEW

**Comment by barbara79** [comment]
https://github.com/commercetools/commercetools-php-sdk/issues/543

File: examples/Store/StoreSetLanguagesAction.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "action": "setLanguages",
+  "languages": ["languagesString"]
```
Comment (katmatt) [inline examples/Store/StoreSetLanguagesAction.json]: Here I would suggest to use a correct value `["en"]` instead of the invalid `["languagesString"]`.

Comment (barbara79) [reply examples/Store/StoreSetLanguagesAction.json]: yes, make sense ;) I'm updating it

**Comment by katmatt** [comment]
Thanks, looks perfect now 🥇

**Pull request merged**